### PR TITLE
Feature/ambassadors-page

### PR DESCRIPTION
**Pull request by misspelz**
image

### CODE REVIEW

**Comment by edustipend-root** [comment]
Hello @misspelz, please check why the build is failing on this PR.
I'll review once the build passes.

**Comment by misspelz** [comment]
> Hello @misspelz, please check why the build is failing on this PR. I'll review once the build passes.

@meekg33k This is the test failing. It has nothing to do with the ambassador's page
image

As for the ambassador's page, I am still waiting for the design

File: src/assets/DeborahAbs.png
Comment (edustipend-root) [inline src/assets/DeborahAbs.png:1]: Why do we have two of these images @misspelz?

File: src/assets/EkiminiAbs.png
Comment (edustipend-root) [inline src/assets/EkiminiAbs.png:1]: Why are we having duplicate of the same images?

File: src/assets/DeborahAbs.png
Comment (misspelz) [reply src/assets/DeborahAbs.png:1]: I have removed all duplicates and pushed

File: src/assets/EkiminiAbs.png
Comment (misspelz) [reply src/assets/EkiminiAbs.png:1]: I have removed all duplicates and pushed

**Comment by edustipend-root** [comment]
What is the difference between this PR and https://github.com/edustipend/dotorg/pull/95 @misspelz?

**Comment by misspelz** [comment]
> What is the difference between this PR and #95 @misspelz?

I was already done with this PR before I saw your message about the large images. So, I created another PR to address that. I think it's best to merge both or what do you think?

**Comment by edustipend-root** [comment]
> > What is the difference between this PR and #95 @misspelz?
> 
> I was already done with this PR before I saw your message about the large images. So, I created another PR to address that. I think it's best to merge both or what do you think?

@misspelz, if the changes in this PR are contained in the other one then I can merge the other one and we close this one.

**Pull request merged**